logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/asset/DeleteAssetUseCase.kt

```diff
@@ -0,0 +1,12 @@
+package com.wire.kalium.logic.feature.asset
+
+import com.wire.kalium.logic.data.asset.AssetRepository
+import com.wire.kalium.logic.data.user.UserAssetId
+
+class DeleteAssetUseCase(private val assetRepository: AssetRepository) {
```

Comment:
In case this use case will have usages on Reloaded, needs documentation and doing it as an interface, so we hide the implementation 😅 otherwise we can mark it as internal 🙌